src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractViolationReporterTest.java

```diff
@@ -133,6 +134,18 @@ public void testCustomMessageWithParametersNegative() throws Exception {
         }
     }
 
+    /*
+      Due to low level configuration setup of AbstractViolationReporter, conventional
+      input validation cannot be done here hence, pure JUnit testing has been
+      done for the time being
+    */
+    @Test
+    public void testSetSeverity() {
+        final EmptyCheck obj = new EmptyCheck();
```

Comment:
`obj` -> 'emptyCheck`, we have no reason to be cryptic in our naming

Reply:
@nrmancuso 
ok fixing it.